axum-extra/Cargo.toml

```diff
@@ -16,7 +16,13 @@ default = ["tracing"]
 
 async-read-body = ["dep:tokio-util", "tokio-util?/io", "dep:tokio"]
 async-stream = [] # unused, remove before the next breaking-change release
-file-stream = ["dep:tokio-util", "tokio-util?/io", "dep:tokio", "tokio?/fs", "tokio?/io-util"]
+file-stream = [
+    "dep:tokio-util",
+    "tokio-util?/io",
+    "dep:tokio",
+    "tokio?/fs",
+    "tokio?/io-util",
+]
```

Comment:
@andrzejSulkowski any reason for these toml file changes? are these only changing the formatting or are there also semantic changes hidden in them?

Reply:
I bet this is automated formatting from zed...